### PR TITLE
Simply documentation in README and reference Poetry cli commands

### PR DESCRIPTION
**Pull request by GiancarloFusiello**
Simply documentation in README and reference Poetry cli commands. Update Makefile according to README changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,59 +24,30 @@ found in the `Makefile` into your command line interface.
...
+To [add](https://poetry.eustace.io/docs/cli/#add) and [remove](https://poetry.eustace.io/docs/cli/#remove)
+a project dependency, run `make shell` to enter the bash shell of the app container
+and make changes as necessary. Any changes made will only update `pyproject.toml`
+and `poetry.lock` files. If you would like to apply the changes to the app container
+then rebuild the container by running `make build`.
 
+Example:
 ```bash
+make shell
+poetry add pytest --dev
```
Comment (giorgiosironi) [inline README.md]: sometimes it's useful to show the prompts here to show the change of context:
```
$ make shell
bash-4.4# ls search/
```
Though it becomes more a sample output than a list of commands.

**Review by giorgiosironi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GiancarloFusiello) [reply README.md]: I did think about adding the prompt. As it's an example . I think I will.

Comment (GiancarloFusiello) [reply README.md]: shell added.

**Review by giorgiosironi — APPROVED** [review]
Wrapper targets being removed to expose directly `poetry`, more flexible approach for running custom commands

**Pull request merged**